Doc/library/asyncio-api-index.rst

```diff
@@ -0,0 +1,207 @@
+.. currentmodule:: asyncio
+
+
+=====================
+High-level APIs Index
+=====================
+
+This page lists all high-level async/await enabled asyncio APIs.
+
+
+Tasks
+=====
+
+Utilities to run asyncio programs, create Tasks, and
+await on multiple things with timeouts.
+
+.. list-table::
+    :widths: 50 50
```

Comment:
Looks pretty good rendered.
<img width="876" alt="screenshot 2018-09-14 14 25 58" src="https://user-images.githubusercontent.com/2680980/45575967-34df8080-b82a-11e8-8684-b7239efcf1a1.png">

Reply:
Adjusting to @ezio-melotti's suggested width would shorten the first entry.

Reply:
This is pretty much what I expected: it has too much whitespace on the first column and leaves less space for the description (e.g. the first description has to take two lines instead of one).
I think the simple table markup also figures out the column width automatically based on the content (and the list-table probably does the same if you omit the width, unless you want consistency between the tables).

Reply:
Screenshot at 30/70
<img width="727" alt="screenshot 2018-09-14 14 34 21" src="https://user-images.githubusercontent.com/2680980/45576265-5856fb00-b82b-11e8-85bd-747856831c87.png">

Reply:
Fair enough.
Just out of curiosity, what did you find awful, and what were you trying to achieve (consistent column width between tables, full width table, others)?
(FWIW, of the two screenshots posted by Carol (thanks!), I prefer the 30/70.)